src/main/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/gencode/GencodeFuncotationFactory.java

```diff
@@ -678,9 +923,32 @@ private static GencodeFuncotation createIntronFuncotation(final VariantContext v
         final Strand strand = Strand.decode( transcript.getGenomicStrand().toString() );
         FuncotatorUtils.assertValidStrand(strand);
 
+        // Get the strand-corrected alleles from the inputs.
+        // Also get the reference sequence for the variant region.
+        // (spanning the entire length of both the reference and the variant, regardless of which is longer).
+        final Allele strandCorrectedRefAllele;
+        final Allele strandCorrectedAltAllele;
+
+        if ( strand == Strand.POSITIVE ) {
+            strandCorrectedRefAllele = variant.getReference();
+            strandCorrectedAltAllele = altAllele;
+        }
+        else {
+            strandCorrectedRefAllele = Allele.create(ReadUtils.getBasesReverseComplement( variant.getReference().getBases() ), true);
+            strandCorrectedAltAllele = Allele.create(ReadUtils.getBasesReverseComplement( altAllele.getBases() ), false);
+        }
+
+        final String referenceBases = FuncotatorUtils.getBasesInWindowAroundReferenceAllele(strandCorrectedRefAllele, strandCorrectedAltAllele, strand, referenceWindow, referenceContext);
+
+        // Set our reference sequence in the Gencode Funcotation Builder:
+        gencodeFuncotationBuilder.setReferenceContext( referenceBases );
+
         // Set as default INTRON variant classification:
         gencodeFuncotationBuilder.setVariantClassification(GencodeFuncotation.VariantClassification.INTRON);
 
+        // Set GC Content:
```

Comment:
Why do you need to do this in a method for setting intron funcotations?

Reply:
@LeeTL1220 GC content is required for splice site variants and since I was calculating it anyway I figured I would include it in all Intron matches, not just the intronic splice sites.

Reply:
ok ... gc content is required for all variants regardless of variant classification.  That was why I was a bit confused.  Is it being rendered for all variants?  If so, then no action needed.

Reply:
Oh, OK.  Yes - GC Content is being calculated for all variants in Gencode.